External/Plugins/ASCompletion/Completion/ASComplete.cs

```diff
@@ -3686,8 +3686,8 @@ static public bool IsTextStyleEx(int style)
             return style == 0 || style == 5 /*word (secondary keywords)*/
                 || style == 10 /*punctuation*/ || style == 11 /*identifier*/ 
                 || style == 16 /*word2 (secondary keywords: class name)*/ 
-                || style == 19 /*globalclass (primary keywords)*/ || style == 23 /*word3 (add keywords3)*/
-                || style == 24 /*word4 (add keywords4)*/ || style == 25 /*word5 (add keywords5)*/
+                || style == 19 /*globalclass (primary keywords)*/ || style == 100 /*word3 (add keywords3)*/
+                || style == 101 /*word4 (add keywords4)*/ || style == 102 /*word5 (add keywords5)*/
```

Comment:
Shouldn't these functions be using the `Lexers.CPP` enum instead of hardcoding the values? :/